### ISSUE TITLE
Refactor workflow for build logs

### ISSUE THREAD

**Issue by justine-gehring** [moderneinc/moderne-cluster-build-logs#11]
From @shanman190:
"More specifically, the failures.csv having the build log at the stage that it does makes it a bit difficult to work with since the rows are so large due to the sheer amount of content. Presently, just deleting the build.log using that side feature, but I'm thinking through if we shouldn't change 01 to be extracting the failures w/o the build.log content, then insert a new 02 that extracts the build log for further data analysis.

So to summarize:
00: download
01: extract failures; no log content
02: extract log content
03: create graphics

User would repeat steps 02 and 03 repeatedly to update the graphics"
